docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -465,3 +467,116 @@ The following overrides are available:
+1. Import the clusters into RHACM as a cluster set, and install Submariner addon on them. For more information, see the section from [1.1.2. Configuring Submariner](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/add-ons/add-ons-overview#configuring-submariner).  
+   Note down the cluster names from the cluster set, these will be used during the cloud provider setup in YugabyteDB Anywhere.
+
+### Prepare Kubernetes clusters for Istio mulit-cluster
+
+Istio service mesh can span across multiple clusters which allows us to configure MCS. It supports different topologies and network configurations. Follow [Install Multicluster](https://istio.io/latest/docs/setup/install/multicluster/) to get started with the multi-cluster Istio setup.
+
+The Istio configuration for each cluster should have following options:
+
+```yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_DNS_CAPTURE: "true"
+        ISTIO_META_DNS_AUTO_ALLOCATE: "true"
+  # rest of the configuration…
+```
+Refer the Istio setup section from [Multi-Region YugabyteDB Deployments on Kubernetes with Istio](https://www.yugabyte.com/blog/multi-region-yugabytedb-deployments-on-kubernetes-with-istio/) for a more step by step guide along with explanation of the options being used.
+
+### Configure the cloud provider for MCS
+
+Once you have the cluster setup ready, you can follow [Configure the Kubernetes cloud provider](#). And refer to this section for region and zone configuration requried for multi-region universes.
+
+
+#### Configure region and zone for GKE MCS
+Set the following values for all the zones from your Kubernetes clusters connected via GKE MCS.
+
+1. Specify all other fields like Region, Zone, etc. by following [Configure region and zones](#configure-region-and-zones).
+1. Set the **Cluster DNS Domain** to `clusterset.local`.
+1. Upload the correct **Kube Config** of the cluster.
+1. Set the **Pod Address Template** to `{pod_name}.<cluster membership name>.{service_name}.{namespace}.svc.{cluster_domain}`, where the `<cluster membership name>` is the membership name of the Kubernetes cluster set during the fleet setup.
+1. Set the **Overrides** as follows:
+   ```yaml
+   multicluster:
+     createServiceExports: true
+     kubernetesClusterId: "<cluster membership name>"
+     mcsApiVersion: "net.gke.io/v1"
+   ```
+For example, if your cluster membership name is `yb-asia-south1`, then the **Add new region** screen would look as follows:
+
+![Add new region screen of YugabyteDB Anywhere with GKE MCS](/images/ee/k8s-setup/k8s-add-region-gke-mcs.png)
```

Comment:
Is the Add region screen in the earlier section out of date?

Reply:
Yes, but in latest version new fields have been added. Should I update that screenshot as part of this PR?

Reply:
I will create a ticket if required. The current screenshot we have in earlier section is rather simple and sufficient. Right now we don't talk about the other options here in this doc as well.